### ISSUE TITLE
SA1504 False Positive on single line accessor with attribute

### ISSUE THREAD

**Issue by elachlan** [DotNetAnalyzers/StyleCopAnalyzers#3434]
When this property is changed to use a single line accessors it still says there is an error
Example:
https://github.com/dotnet/msbuild/blob/fb700f90493a0bf47623511edf28b1d6c114e4fa/src/Build/Instance/ProjectInstance.cs#L1081-L1091

Manual Fix causes error:
```
        /// <summary>
        /// Sets the initial targets
        /// Only set during evaluation, so does not check for immutability.
        /// </summary>
        List<string> IEvaluatorData<ProjectPropertyInstance, ProjectItemInstance, ProjectMetadataInstance, ProjectItemDefinitionInstance>.InitialTargets
        {
            [DebuggerStepThrough]
            get { return InitialTargets; }
            set { InitialTargets = value; }
        }
```

Removing the `[DebuggerStepThrough]` Attribute fixes the issue.

**Comment by sharwell** [comment]
Is an error still reported if you expand it?

```cs
List<string> IEvaluatorData<ProjectPropertyInstance, ProjectItemInstance, ProjectMetadataInstance, ProjectItemDefinitionInstance>.InitialTargets
{
    [DebuggerStepThrough]
    get
    {
        return InitialTargets;
    }

    set
    {
        InitialTargets = value;
    }
}
```

**Comment by elachlan** [comment]
Yes, the expansion works.

**Comment by elachlan** [comment]
@sharwell this analyzer also deleted comments inside the properties. See https://github.com/dotnet/msbuild/pull/7270 for examples.
